scripts/validate-changelog.sh

```diff
@@ -8,9 +8,10 @@ if [[ $# -eq 0 ]]; then
 fi
 
 package_name="$1"
+shift  # remove package name from arguments
 
 if [[ "${GITHUB_REF:-}" =~ '^release/' ]]; then
-  yarn auto-changelog validate --tag-prefix "${package_name}@" --rc
+  yarn auto-changelog validate --prettier --tag-prefix "${package_name}@" --rc "$@"
```

Comment:
The `"$@"` allows us to run this script with `--fix`.

Reply:
thank you! 😄